pkg/telemetry/tm_test.go

```diff
@@ -16,6 +17,10 @@ var (
 )
 
 func TestMetrics(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("broken")
```

Comment:
bummer about all of these. for grep-ability, mind adding `// TODO` or something on the skipped tests?

Reply:
Well, when you run tests with -v it will show that something was skipped. As for grepping ".Skip". But, I can add TODO notes.